### DISCUSSION TITLE
Switched MacBooks, Temperatures Not Displaying

### DISCUSSION THREAD

**Discussion by HilloBeans** [artisan-roaster-scope/artisan#1573] [Q&A]
Hi Everyone,

I just switched to a new MacBook and am trying to do a. first roast using this computer.  Artisan (2.10.4) seems to run fine, except that temperatures are not reading or displaying.  Trying to attach a screenshot here.

Roaster is a Diedrich IR2.5

Can someone please help me with a solution?

Thanks in advance for any assistance you can offer.

Kent

<img width="479" alt="Screenshot 2024-05-30 at 2 53 31 PM" src="https://github.com/artisan-roaster-scope/artisan/assets/89611011/bd26c302-ab04-4943-8858-2ac493050ed4">

**Comment by CarefreeBuzzBuzz** [comment]
Need more info.  How did you do the switch.  Did you copy your settings over?  Do you have the correct drivers if any needed for your machine installed?  
----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Support Artisan RoasterScope  <https://artisan-scope.org/>by using 
Artisan.Plus <https://artisan.plus/en/>

**Comment by HilloBeans** [comment]
I used Apple's Migration Assistant to copy my old laptop to this one - and
all passwords, documents, etc. came over intact.  When I saw this problem,
I downloaded the most recent version of Artisan and also the Phidget's for
Mac.  I didn't download or install any other drivers.

**Comment by CarefreeBuzzBuzz** [comment]
Are you sure your settings file came over?  Did you load those settings?  

Did you review this page and make sure you copied things over?

If you open the Phidgets control panel are there temp readings?  



https://artisan-scope.org/devices/phidgets/￼
Phidgets
artisan-scope.org


----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Support Artisan RoasterScope  <https://artisan-scope.org/>by using 
Artisan.Plus <https://artisan.plus/en/>

**Comment by HilloBeans** [comment]
Thanks for these suggestions - I went to that control panel and found that,
though I had allowed Phidgets in the security settings, I needed to do it
again.  So now all is well.

I appreciate your help and fast responses here.
